main.go

```diff
@@ -12,11 +12,33 @@ import (
 // JSON example
 // {"ar": ["joaothallis"], "elixir": ["josevalim", "eksperimental"]}
 
+func getPrarFilePath(globalArg string) (string, error) {
+	switch globalArg {
+	case "--global":
```

Comment:
Is that possible to refactor to use `flag.Bool` as in this example: https://gobyexample.com/command-line-flags

Reply:
I applied the flag library, and I refactor this main function